Python/Audio_Format_Converter/Audio_File_Converter.py

```diff
@@ -0,0 +1,45 @@
+# Script to Convert audio files from one format to another
+import os
+from glob import glob
+import pydub
+import mutagen
+
+# Directory containing the audio File
+song_dir = 'C:\\Users\\asus\\Desktop\\Songs\\*.mp3'  # Extension is according the the file type before conversion.
+
+
+def conversion(song):
+    for song in song:
+        mp3_song = os.path.splitext(song)[0] + '.wav'  # Extension in which file is supposed to be convert
+        """
+        Support .mp3,
+                .mp4,
+                .ogg,
+                .aac,
+                .flv,
+                .mkv,
+                .flac,
+                .wav
+        """
+        sound = pydub.AudioSegment.from_mp3(song)
+        sound.export(mp3_song, format="wav")  # Change the extension to the desired format
```

Comment:
Can you try putting the comment above the Line of Code. You can follow the same for all other Comments as well.

Reply:
Updated in the latest commit